2-mandatory/1-weather-report.js

```diff
@@ -11,8 +11,15 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
+
```

Comment:
This looks good to me 👍 2 small notes:
- Sometimes it's more readable to use template literals in JavaScript. Read more about this here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
- Keep an eye on your code indentation (spacing). This can make your code easier to read for both yourself and other developers.